### PR TITLE
New permissions

### PR DESCRIPTION
**Pull request by ericgaspar**
- *New permissions*

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -143,12 +154,12 @@ then
...
-    	# ynh_exec_as $app env PATH=$PATH drush @$app pm-update -y drupal
-    	# ynh_exec_as $app env PATH=$PATH drush @$app updatedb -y
-    	ynh_exec_as $app env PATH=$PATH drush @$app cache-clear all 
-    	ynh_exec_as $app env PATH=$PATH drush @$app variable-set --exact maintenance_mode 0 
+    	sudo -u $app env PATH=$PATH drush @$app variable-set --exact maintenance_mode 1 
+    	sudo -u $app $app env PATH=$PATH drush @$app cache-clear all 
+    	# sudo -u $app env PATH=$PATH drush @$app pm-update -y drupal
+    	# sudo -u $app env PATH=$PATH drush @$app updatedb -y
+    	sudo -u $app $app env PATH=$PATH drush @$app cache-clear all 
+    	sudo -u $app $app env PATH=$PATH drush @$app variable-set --exact maintenance_mode 0
```
Comment (alexAubin) [inline scripts/upgrade:162]: Duplicated $app ?

```suggestion
    	sudo -u $app env PATH=$PATH drush @$app variable-set --exact maintenance_mode 1 
    	sudo -u $app env PATH=$PATH drush @$app cache-clear all 
    	# sudo -u $app env PATH=$PATH drush @$app pm-update -y drupal
    	# sudo -u $app env PATH=$PATH drush @$app updatedb -y
    	sudo -u $app env PATH=$PATH drush @$app cache-clear all 
    	sudo -u $app env PATH=$PATH drush @$app variable-set --exact maintenance_mode 0 
```

**Review by alexAubin — COMMENTED** [review]
(LGTM apart from oddish duplicated $app)

File: scripts/upgrade (hunk 1 above)
Comment (ericgaspar) [reply scripts/upgrade:162]: yes of course 🙄

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**